pkgs/top-level/all-packages.nix

```diff
@@ -31972,6 +31972,12 @@ with pkgs;
 
   please-cli = callPackage ../applications/misc/please-cli { };
 
+  plemoljp-font = callPackage ../data/fonts/plemoljp { };
```

Comment:
```
- Attribute `pkgs.plemoljp-font` is a new top-level package using `pkgs.callPackage ./pkgs/data/fonts/plemoljp { /* ... */ }`.
  Please define it in pkgs/by-name/pl/plemoljp-font/package.nix instead.
  See `pkgs/by-name/README.md` for more details.
  Since the second `callPackage` argument is `{ }`, no manual `callPackage` in pkgs/top-level/all-packages.nix is needed anymore.
```

I didn't know we should prefer by-name style even if adding fonts. I'll update

Reply:
yes those need to be added by-name